### PR TITLE
new-feature:add k3zkutil

### PR DESCRIPTION
**Pull request by IloveKanade**
The k3zkutilmodified according to the specification

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+logger = logging.getLogger(__name__)
+
+# try to load config from a top-level module.
+try:
+    import pykitconfig
+except ImportError:
+    pykitconfig = object()
+    logger.info('pykitconfig not found by "import pykitconfig".'
+                ' Using default config.'
+                ' You can create file "pykitconf.py" to define default config for pykit.')
```
Comment (drmingdrmer) [inline config.py]: We need to replace this with k3confloader...
https://github.com/pykit3/k3confloader

Maybe a new test should be added to ensure it reads from the expected config file.

Comment (IloveKanade) [reply config.py]: > We need to replace this with k3confloader...
> https://github.com/pykit3/k3confloader
> 
> Maybe a new test should be added to ensure it reads from the expected config file.

okay, I see!

Comment (IloveKanade) [reply config.py]: > We need to replace this with k3confloader...
> https://github.com/pykit3/k3confloader
> 
> Maybe a new test should be added to ensure it reads from the expected config file.

done
I seem to see that zkconf already has ut for conf, do I need to add something else?

Comment (drmingdrmer) [reply config.py]: > zkconf already has ut for conf
What do you mean the "has ut for conf"?

Comment (IloveKanade) [reply config.py]: > > zkconf already has ut for conf
> > What do you mean the "has ut for conf"?

I mean zkconf already has ut to test reads from the expected Config file, such as test_zkconf.test_default, do I need to add some more?

Comment (drmingdrmer) [reply config.py]: Hm... sorry I missed that. Then there won't be a problem.

**Review by drmingdrmer — APPROVED** [review]
LGTM

**Pull request merged**